bundles/automation/org.eclipse.smarthome.automation.module.core/src/main/java/org/eclipse/smarthome/automation/module/core/factory/CoreModuleHandlerFactory.java

```diff
@@ -182,6 +183,8 @@ protected synchronized ModuleHandler internalCreate(final Module module, final S
                 return postCommandActionHandler;
             } else if (RuleEnablementActionHandler.UID.equals(moduleTypeUID)) {
                 return new RuleEnablementActionHandler((Action) module, ruleRegistry);
+            } else if (RunRuleActionHandler.UID.equals(moduleTypeUID)){
```

Comment:
If you could add a space before the opening bracket, I would be fine to merge the PR :-)